README.md

```diff
@@ -55,6 +55,25 @@ other: FFF
 other: GGG
 ```
 
+Additionally, if there is a repository in the org named `.github`, it will be
+used as a base repository if none is specified.
```

Comment:
Do you think you could add a config example for how this works? Would it just be files without the` _extends`?

Reply:
The config example below is does that...is there something more you would like to see in it?

Reply:
would this work for personal accounts, too? Say I’d create `gr2m/.github` to see defaults for apps that I install, would that work?